### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vthg2themax**
Create a section to help for new people building this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,7 +176,37 @@ function run_disown_silence(){
...
+sudo apt install libgit2-dev
+sudo apt install cmark
+sudo apt install git
+sudo apt install libssh2
+sudo apt install openssl
+sudo apt install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools
+sudo apt install qttools5-dev
+sudo apt install ninja-build
+#Change to your gittyup directory below
+cd ~/Nextcloud/Projects/gittyup
```
Comment (Murmele) [inline README.md:194]:
```suggestion
cd <path to repository>/gittyup
```

File: README.md [hunk 2]
```diff
@@ -176,7 +176,37 @@ function run_disown_silence(){
...
+git fetch
+git submodule init
+git submodule update
+git pull
+git checkout deps
+cd dep/openssl/openssl/
+./config -fPIC
+make
+#Change to your gittyup directory below
+cd ~/Nextcloud/Projects/gittyup
```
Comment (Murmele) [inline README.md:204]:
```suggestion
cd <path to repository>/gittyup
```

**Review by Murmele — CHANGES_REQUESTED** [review]

**Comment by Murmele** [comment]
Hi, there is already a how to build section, why do you need an extra one?

**Comment by vthg2themax** [comment]
This just makes it easy to create a build.sh file to automatically let a new user who downloads the source to quickly create a new executable on Linux. Please let me know if that's not something you want, or if it should be put somewhere else.

**Comment by Bian-Sh** [comment]
收到

**Comment by Murmele** [comment]
@vthg2themax Can you add a script to the pack folder and refering in the Readme to that script? So a user can directly download it and creating a package

**Review by vthg2themax — COMMENTED** [review]
Looks good to me. Let me know if you have other thoughts.

File: README.md [hunk 3]
```diff
@@ -132,8 +132,12 @@ where `<path-to-qt>` points to the Qt install directory that contains
 `bin`, `lib`, etc.
 
 **Build**
-
+```
     ninja
+```
+    
+### A Convenient Shell Script for Linux is available in the following location:
```
Comment (Murmele) [inline README.md:139]:
```suggestion
### A Convenient Shell Script for Ubuntu is available [here](https://raw.githubusercontent.com/Murmele/Gittyup/master/pack/buildUbuntu.sh), and will install all the necessary prerequisites, and build a release version for immediate use.
```

File: README.md [hunk 4]
```diff
@@ -132,8 +132,12 @@ where `<path-to-qt>` points to the Qt install directory that contains
...
+`pack/build.sh`, and will install all the necessary prerequisites, and build a release version for immediate use.
```
Comment (Murmele) [inline README.md:140]:
```suggestion
```

File: README.md (hunk 3 above)
Comment (Murmele) [reply README.md:139]: What do you think about this? Please rename build.sh to buildUbuntu.sh

**Review by Murmele — CHANGES_REQUESTED** [review]

**Comment by vthg2themax** [comment]
@Murmele I think your edits are wonderful. Thank you! Let me know if there's anything further that should be modified.

**Comment by vthg2themax** [comment]
@Murmele This link appears to not work:
https://raw.githubusercontent.com/Murmele/Gittyup/master/pack/buildUbuntu.sh
Is that something that will be created later, or do I need to rename something?

**Comment by Murmele** [comment]
> @Murmele This link appears to not work: https://raw.githubusercontent.com/Murmele/Gittyup/master/pack/buildUbuntu.sh Is that something that will be created later, or do I need to rename something?

@vthg2themax the link will work as soon it is merged, but you have to rename the shell script to buildUbuntu.sh ;)

**Comment by vthg2themax** [comment]
I renamed the file. Thank you for the reminder!

**Review by vthg2themax — COMMENTED** [review]
I think this should do it

File: pack/Readme.md [hunk 5]
```diff
@@ -1,2 +1,3 @@
 Flatpak manifest can be found at: https://github.com/flathub/com.github.Murmele.Gittyup
 Arch PKGBUILD file can be found at: TODO (currently it is in #314)
+Linux (Tested On Ubuntu 22.04LTS) version is available in the build.sh file included in this directory.
```
Comment (Murmele) [inline pack/Readme.md:3]:
```suggestion
Linux (Tested On Ubuntu 22.04LTS) version is available in the buildUbuntu.sh file included in this directory.
```

**Review by Murmele — CHANGES_REQUESTED** [review]

**Review by Murmele — APPROVED** [review]

**Pull request merged**